clarity-ext-scripts/scripts/reportCOVID19samples.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Dmitry Chekmenev, 2020
+# Cron job which extract all samples with their UDFs of Covid-19 Clarity LIMS Project
+
+set -euo pipefail
+
+current_date=`date +%Y%m%d`
+echo "$(date) Running Clarity sample data export for covid19 on ${current_date}"
+
+psql --no-align --field-separator=',' --username='clarity' --dbname='ClarityDB' --file='/opt/gls/clarity/users/glsai/deployment/claritylims/clarity-ext-scripts/scripts/reportCOVID19samples.sql' --output="/tmp/report_$(current_date)"
```

Comment:
I still think we need `--pset footer` in the command, otherwise we get the final line with the number of records printed at the end of the output (which, in turn, screws up the datagraphics parsing script).